demisto_sdk/commands/validate/file_validator.py

```diff
@@ -674,18 +680,43 @@ def run_all_validations_on_file(self, file_path: str, file_type: str = None) ->
                         'Layouts and Descriptions')
             self._is_valid = False
 
-    def validate_all_files(self):
+    def validate_all_files(self, skip_conf_json):
         print('Validating all files')
-        print('Validating conf.json')
-        conf_json_validator = ConfJsonValidator()
-        if not conf_json_validator.is_valid_conf_json():
-            self._is_valid = False
+
+        if not skip_conf_json:
+            print('Validating conf.json')
+            conf_json_validator = ConfJsonValidator()
+            if not conf_json_validator.is_valid_conf_json():
+                self._is_valid = False
+
         packs = {os.path.basename(pack) for pack in glob(f'{PACKS_DIR}/*')}
         self.validate_pack_unique_files(packs)
         all_files_to_validate = set()
-        for directory in [PACKS_DIR, BETA_INTEGRATIONS_DIR, TEST_PLAYBOOKS_DIR]:
-            all_files_to_validate |= {file for file in glob(fr'{directory}/**', recursive=True) if
-                                      not should_file_skip_validation(file)}
+
+        # go over packs
+        for pack_name in os.listdir(PACKS_DIR):
```

Comment:
what about TEST_PLAYBOOKS_DIR ?

Reply:
Will not exist next week. All test playbooks should be in packs